src/app/loaders/express/cors.ts

```diff
@@ -0,0 +1,19 @@
+import cors from 'cors'
+
+import { app, corsWhiteList } from '../../config/config'
+
+export const corsMiddleware = () => {
+  return cors({
+    origin: function (origin, callback) {
+      // If no origin or from self, disable CORS
+      if (!origin || app.appUrl === origin) {
+        callback(null, false)
```

Comment:
do you think this would be clearer if we just create one boolean which explains all the cases? e.g.
```
const isOriginAllowed = 
  // allow server-to-server requests
  !origin ||
  // allow requests from own origin
  app.appUrl === origin ||
  // allow requests from whitelisted domains
  corsWhiteList.indexOf(origin) !== -1
callback(null, isOriginAllowed)
```

Reply:
might need to flip some of the booleans depending on my other comment

Reply:
will defer and use this, probably fine to set origin even when from our own domain